org.squidowl.halloy.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <content_rating type="oars-1.0">
+    <content_attribute id="social-chat">moderate</content_attribute>
+  </content_rating>
+  <id>org.squidowl.halloy</id>
```

Comment:
Is `squidowl.org` under the control of the developers?

Reply:
Yes, we've recently acquired the domain but we don't have a website up yet.